services/order-service/.env.example

```
@@ -0,0 +1,32 @@
+# 하위 서비스 URL (docker-compose 네트워크 내 서비스명)
+PRODUCT_SERVICE_URL=http://product-service:8000
+PAYMENT_SERVICE_URL=http://payment-service:8000
+
+# NATS 브로커
+NATS_URL=nats://nats:4222
+
+# 서비스 간 HTTP 호출 타임아웃 (초, float)
+HTTP_TIMEOUT_SECONDS=5.0
+
+# NATS 연결 타임아웃 (초, int)
+NAT_CONNECT_TIMEOUT_SECONDS = 5
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)